code/datums/station_traits/positive_traits.dm

```diff
@@ -108,7 +108,7 @@
 /datum/station_trait/filled_maint
 	name = "Filled up maintenance"
 	trait_type = STATION_TRAIT_POSITIVE
-	weight = 5
+	weight = 6
```

Comment:
I see no reason to change anything in this file

Reply:
i. e. I went over the changes you made in this file and don't see reason for any of them

Reply:
The most important one is Bubber's emphasis on medical; having a deathrattle more commonly apt for medical personnel can really save a round when it's most needed, everything else is soft rebalancing.

Frankly i'd want a weight of 3-4 for medical deathrattles.